ouroboros-consensus/src/Ouroboros/Consensus/BlockchainTime/WallClock/HardFork.hs

```diff
@@ -42,7 +42,7 @@ hardForkBlockchainTime registry
                        cfg
                        getLedgerState = do
     run <- HF.runWithCachedSummary (summarize <$> getLedgerState)
-    waitUntilSystemStart tracer time
```

Comment:
Don't ask for a tracer of `TraceBlockchainTimeEvent`, but instead ask for one that uses `RelativeTime`, and then change the tracing type back to use `UTCTime`.